tests/cli_test.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+from os import system
+
+DIR = Path(__file__).absolute().parents[0]
```

Comment:
```suggestion
DIR = Path(__file__).resolve().parent
```

`absolute()` is an undocumented function, `.resolve()` is better. And `.parent` is better than `.parents[0]`.